bin/clean-aws.js

```diff
@@ -187,20 +188,29 @@ function cleanupOldItems(apiInfo, applyChanges = false) {
     .then(items => {
       if (items.length > 0) {
         if (applyChanges) {
+          if (apiInfo.limit && items.length > apiInfo.limit) {
+            console.log(
+              `There are ${items.length} items eligible for deletion but the AWS service in question is limited to delete ${apiInfo.limit} at once. I am going to delete ${apiInfo.limit} right now. Run the script again to delete more.`
+            );
+            // arbitrarily delete all item after position apiInfo.limit
+            items.splice(apiInfo.limit);
+          }
```

Comment:
When trying some changes locally, I ran into 

```
Error deleting 94 items: LimitExceededException: Subscriber limit exceeded: Only 50 tables can be created, updated, or deleted simultaneously
    at Request.extractError (/Users/bastian/instana/code/nodejs/node_modules/aws-sdk/lib/protocol/json.js:52:27)
    at Request.callListeners (/Users/bastian/instana/code/nodejs/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
```

for DynamoDB, so I added this limit.